website/docs/r/opc_database_service_instance.html.markdown

```diff
@@ -0,0 +1,168 @@
+
+`usable_storage` - (Required) Storage size for data (in GB). Minimum value is 15. Maximum value depends on the backup destination: if BOTH is specified, the maximum value is 1200; if OSS or NONE is specified, the maximum value is 2048.
+
+* `cloud_storage_container` - (Optional) Name of the Oracle Storage Cloud Service container used to provide storage for your service instance backups. Use the following format to specify the container name: <storageservicename>-<storageidentitydomain>/<containername>
+
+* `cloud_storage_username` - (Optional) Username for the Oracle Storage Cloud Service administrator.
+
+* `cloud_storage_password` - (Optional) Password for the Oracle Storage Cloud Service administrator.
+
+* `create_storage_container_if_missing` - (Optional) Specify if the given cloud_storage_container is to be created if it does not already exist. Valid values are true and false. Default value is false.
+
+* `disaster_recovery` - (Optional) Specify if an Oracle Data Guard configuration is created using the Disaster Recovery option or the High Availability option. Valid values are yes and no. Default value is no.
+
+* `failover_database` - Specify if an Oracle Data Guard configuration comprising a primary database and a standby database is created. Valid values are yes and no. Default value is no.
+
+* `golden_gate` - (Optional) Specify if the database should be configured for use as the replication database of an Oracle GoldenGate Cloud Service instance. Valid values are yes and no. Default value is no. You cannot set goldenGate to yes if either isRac or failoverDatabase is set to yes.
+
+* `ibkup` - (Optional) Specify if the service instance's database should, after the instance is created, be replaced by a database stored in an existing cloud backup that was created using Oracle Database Backup Cloud Service. Valid values are yes and no. Default value is no.
+
+* `ibkup_cloud_storage_username` - (Optional)
+Username of the Oracle Cloud user. This parameter is required if ibkup is set to yes.
+
+* `ibkup_cloud_storage_password` - (Optional)
+Password of the Oracle Cloud user specified in ibkup_cloud_storage_user. This parameter is required if ibkup is set to yes.
+
+* `ibkup_database_id` - (Optional) Database id of the database from which the existing cloud backup was created. This parameter is required if ibkup is set to yes.
+
+* `ibkup_decryption_key` - (Optional) Password used to create the existing, password-encrypted cloud backup. This password is used to decrypt the backup. Specify either ibkup_decryption_key or ibkup_wallet_file_content for decrypting the backup. This parameter is required if ibkup is set to yes.
+
+* `ibkup_wallet_file_content` - (Optional) String containing the xsd:base64Binary representation of the cloud backup's wallet file. This wallet is used to decrypt the backup. Specify either ibkup_decryption_key or ibkup_wallet_file_content for decrypting the backup. This parameter is required if ibkup is set to yes.
```

Comment:
In other clouds we handle the base64 encoding if it’s not already base64 encoded